docs/spec/type_system.rst

```diff
@@ -244,6 +246,35 @@ We cannot treat timestamps for different timezones as the same time because the
 semantic meaning. We also cannot treat timestamps with different unit types as same since they all have very different
 ranges. So, no normalization is implemented for timestamps.
 
+.. note::
+
+  For compatibility reasons, `kartothek` coerces timestamps to `us` accuracy, effectively truncating the timestamp. If the timestamp actually has a higher accuracy, arrow raises an exception, rejecting it
```

Comment:
Compatibility with what? hive?

Also, a period or colon is missing at the end of this line.

Reply:
not hive. to be 100% honest, I do not know anymore. I believe this is connected to parquet itself and I merely know that it will/is possible with arrow 3.0
iirc, different java implementations are struggling with this.

I simply cannot put more information into this